products/bash/release-datapower.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+#******************************************************************************
+# Licensed Materials - Property of IBM
+# (c) Copyright IBM Corporation 2020. All Rights Reserved.
+#
+# Note to U.S. Government Users Restricted Rights:
+# Use, duplication or disclosure restricted by GSA ADP Schedule
+# Contract with IBM Corp.
+#******************************************************************************
+
+#******************************************************************************
+# PREREQUISITES:
+#   - Logged into cluster on the OC CLI (https://docs.openshift.com/container-platform/4.4/cli_reference/openshift_cli/getting-started-cli.html)
+#
+# PARAMETERS:
+#   -n : <namespace> (string), Defaults to "cp4i"
+#   -r : <release-name> (string), Defaults to "datapower"
+#   -p : indicates production mode
+#   -a : admin password, defaults to "admin"
+#
+# USAGE:
+#   With defaults values
+#     ./release-datapower.sh
+#
+#   Overriding the namespace and release-name
+#     ./release-datapower -n cp4i-prod -r datapower -p -a admin
+
+function usage {
+    echo "Usage: $0 -n <namespace> -r <release-name> [-t]"
+}
+
+namespace="cp4i"
+release_name="datapower"
+production="false"
+admin_password="admin"
+flavour="developers-limited"
+memory_limit="4Gi"
+replicas=1
+
+while getopts "n:r:pa:" opt; do
+    case ${opt} in
+        n ) namespace="$OPTARG"
+        ;;
+        r ) release_name="$OPTARG"
+        ;;
+        p ) production="true"
+        ;;
+        a ) admin_password="$OPTARG"
+        ;;
+        \? ) usage; exit
+        ;;
+    esac
+done
+
+
+if [[ "$production" == "true" ]]
+then
+    echo "Production Mode Enabled"
+    flavour="production"
+    memory_limit="8Gi"
+    replicas=5
+fi
+
+# Create ConfigMap for default
+oc create configmap -n ${namespace} default-config --from-file=datapower/default.cfg
+
+# Create ConfigMap for test domain
+oc create configmap -n ${namespace} test-config --from-file=datapower/testconfig.cfg
+
+# Create ConfigMap for test domain local
+oc create configmap -n ${namespace} test-tar --from-file=datapower/test.tar.gz
+
+# Create Secret with certificate
+oc create secret generic -n ${namespace} jon --from-file=datapower/sharedcerts/jon.ssk
+
+# Create Secret with DataPower admin credentials
+oc create secret generic -n ${namespace} datapower-admin-credentials --from-literal=password=${admin_password}
+
+# Create DataPowerService
+cat << EOF | oc apply -f -
```

Comment:
This path to the `datapower` dir wlll only work if the current working dir is the same as where this script is located (`products/bash`). I suggest getting the script dir using:
```
SCRIPT_DIR=$(dirname $0)
```
And then using `${SCRIPT_DIR}/datapower/...`

Reply:
thanks! I have adapted the script